bokehjs/src/coffee/renderer/glyph/circle.coffee

```diff
@@ -151,6 +151,39 @@ define [
         .value()
       return hits
 
+    _hit_span: (geometry) ->
+      [vx, vy] = [geometry.vx, geometry.vy]
+
+      if geometry.direction == 'v'
+        yr = @renderer.yrange()
+        y0 = yr.attributes.start
+        y1 = yr.attributes.end
```

Comment:
Yes, right. It's old code from my self teaching mess.. :-)  should have already changed